src/reanimated2/animation/spring.ts

```diff
@@ -68,19 +68,19 @@ export const withSpring = ((
       animation: InnerSpringAnimation,
       now: Timestamp
     ): boolean {
-      const { toValue, startTimestamp, current } = animation;
+      const { innerToValue, startTimestamp, current } = animation;
```

Comment:
Shouldn't it be `toValue: innerToValue` instead? 🤯 

As you can see, renaming variables in existing code can lead to potential bugs.

Finally, why this regression has not been caught by any of our unit tests?

Reply:
My guess would be because typeof animation is tragic 🤣